lib/msf/core/post/file.rb

```diff
@@ -880,7 +880,7 @@ def _win_bin_write_file(file_name, data, chunk_size = 5000)
     b64_filename = "#{file_name}.b64"
     begin
       _win_ansi_write_file(b64_filename, b64_data, chunk_size)
-      cmd_exec("certutil -decode #{b64_filename} #{file_name}")
+      cmd_exec("certutil -f -decode #{b64_filename} #{file_name}")
```

Comment:
There's another certutil call here. Although there is potentially an edgecase where the temp decoded b64 file already exists, i'll leave that out for now.